test/t8_cmesh/t8_gtest_cmesh_vertex_conn_tree_to_vertex.cxx

```diff
@@ -0,0 +1,370 @@
+  }
+
+  t8_cmesh_t cmesh;
+};
+
+/** Check attribute values of cmeshes against reference values. */
+TEST_P (cmesh_vertex_conn_ttv_with_cmesh_functions, DISABLED_get_global)
+{
+  const t8_locidx_t num_local_trees = t8_cmesh_get_num_local_trees (cmesh);
+  for (t8_locidx_t itree = 0; itree < num_local_trees; ++itree) {
+
+    const t8_eclass_t tree_class = t8_cmesh_get_tree_class (cmesh, itree);
+    const int num_tree_vertices = t8_eclass_num_vertices[tree_class];
+
+    /* Get all vertices */
+    const t8_gloidx_t *global_vertices = t8_cmesh_get_global_vertices_of_tree (cmesh, itree, num_tree_vertices);
+
+    const t8_gloidx_t start_index = itree * T8_ECLASS_MAX_CORNERS;
+    for (int ivertex = 0; ivertex < num_tree_vertices; ++ivertex) {
+      /* Get the stored global vertex id */
+      const t8_gloidx_t global_vertex = t8_cmesh_get_global_vertex_of_tree (cmesh, itree, ivertex, num_tree_vertices);
+      /* Check value */
+      EXPECT_EQ (global_vertex, start_index + ivertex);
+      EXPECT_EQ (global_vertices[ivertex], start_index + ivertex);
+    }
+  }
+}
+
+/* TODO: Remove the tests belonging to cmesh_vertex_conn_ttv_temp
+ *       as soon as we can enable the tests cmesh_vertex_conn_ttv.
+ *       That is as soon as we can add attributes to cmeshes while deriving. */
+
+#define VTT_TEST_MAX_NUM_TREES 100
+
+class cmesh_vertex_conn_ttv_with_cmesh_functions_temp:
+  public testing::TestWithParam<std::tuple<t8_gloidx_t, t8_eclass_t>> {
+ protected:
+  void
+  SetUp () override
+  {
+    num_trees = std::get<0> (GetParam ());
+    const t8_eclass_t tree_class = std::get<1> (GetParam ());
+    t8_cmesh_init (&cmesh);
+
+    t8_debugf ("Testing cmesh with %li trees of class %s\n", num_trees, t8_eclass_to_string[tree_class]);
+
+    for (t8_locidx_t itree = 0; itree < num_trees; ++itree) {
+      /* Set this tree's class. */
+      t8_cmesh_set_tree_class (cmesh, itree, tree_class);
+      /* Allocate space for entries. */
+      const int num_tree_vertices = t8_eclass_num_vertices[tree_class];
+      t8_gloidx_t *global_indices = T8_ALLOC (t8_gloidx_t, num_tree_vertices);
+      /* Fill with i, i+1, i+2, i+3, ... */
+      const t8_gloidx_t start_index = itree * T8_ECLASS_MAX_CORNERS;
+      for (t8_locidx_t ientry = 0; ientry < num_tree_vertices; ++ientry) {
+        global_indices[ientry] = start_index + ientry;
+      }
+      t8_cmesh_set_global_vertices_of_tree (cmesh, itree, global_indices, num_tree_vertices);
+      /* It is save to free the entries after commit, since the value got copied. */
+      T8_FREE (global_indices);
+    }
+
+    /* Commit the cmesh */
+    t8_cmesh_commit (cmesh, sc_MPI_COMM_WORLD);
+  }
+
+  void
+  TearDown () override
+  {
+    t8_cmesh_destroy (&cmesh);
+  }
+
+  t8_cmesh_t cmesh;
+  t8_gloidx_t num_trees;
+};
+
+/** Check for correct ttv entries. */
+#if 0
+// Reactive this line when we enable the tests with derived attributes
+TEST_P (cmesh_vertex_conn_ttv, DISABLED_get_global)
```

Comment:
Why is the DISABLED keyword not working?

Reply:
I dont know why it wasnt properly working. Will check again.